alliance.md

```diff
@@ -4,22 +4,21 @@ css_id: alliance
 ---
 
 
-# The Uptane Alliance
+# Uptane: A JDF/Linux Foundation Project
 
 The Uptane Alliance was formally established in July of 2018 for the express
 purpose of standardizing an open and secure design for over-the-air updates
 for automobiles.  In practice, this process has involved improving and
 standardizing the Uptane open-source software. Formed under the umbrella of
-IEEE's [International Standards and Technology Organization](https://ieee-isto.org/) and now 
-a [Linux Foundation Joint Development Foundation](http://www.jointdevelopment.org/) alliance, the
-Alliance is a nonprofit consortium of companies, non-profit groups, and government agencies.
+IEEE's [International Standards and Technology Organization](https://ieee-isto.org/), the legal name of the initiative is now the Joint Development Foundation
+Projects, LLC, Uptane Series, and it is hosted by the  
```

Comment:
The legal name is not entirely clear to me. Is it "JDF Project, LLC, Uptane Series" or "JDF Projects, LLC Uptane"?

Reply:
I'm going to check and change. Good catches.

Reply:
Fixed